configuration/encountertypes/encounterTypes.csv

```diff
@@ -61,3 +61,4 @@ ${encounterType.RADIOLOGY_STUDY.uuid},,Examen de radiologie,Radiology Study - re
 ${encounterType.ATTACHMENT_UPLOAD.uuid},TRUE,Attachment Upload,Encounters used to record uploads of attachments.
 ${encounterType.REHAB_EVAL.uuid},,Rehab Evaluation,A rehabilitation encounter with an initial or ongoing evaluation
 ${encounterType.NURSE_CONSULT.uuid},,Nurse Consult,A consult with a nurse
+${encounterType.PERU_PRESCRIPTION.uuid},,Peru Prescription,Drugs prescription made for COR
```

Comment:
Let's rename this to just "Prescription"... it will only be used in Peru for now (and perhaps forever) but that's fine.